Gopkg.toml

```diff
@@ -35,6 +35,10 @@
   name = "github.com/golang/protobuf"
   version = "1.1.0"
 
+[[constraint]]
+  name = "github.com/status-im/go-fcm"
+  revision = "024ca6a2c5444c93980f558f91c35a2defebd362"
```

Comment:
it was possible to keep modules at the same path, like for geth, not sure if it makes sense 

```
   name = "github.com/ethereum/go-ethereum"
   version = "=v1.8.17"
   source = "github.com/status-im/go-ethereum"
```